src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
+
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, uint wParam, int lParam);
```

Comment:
Good point. In fact SendMessage might send strings in *both* encodings (UTF8 and UTF16) so defining a `string` or `StringBuilder` overload here in PInvoke would force only one encoding to be used. Offering raw pointers, as the native code does, is probably the safest option.